src/write/coff/object.rs

```diff
@@ -257,6 +260,7 @@ impl<'a> Object<'a> {
             },
             Architecture::Aarch64 => match typ {
                 coff::IMAGE_REL_ARM64_SECTION => Some(16),
+                coff::IMAGE_REL_ARM64_BRANCH26 => Some(26),
```

Comment:
Do you actually need this? This is used in `write::Object::write_relocation_addend`, and it can't handle a size of 26.

Reply:
No, the only function that I need updated is `coff_translate_relocation`, but I assumed that the other functions need to be updated to match.

Should I also undo the change to ` CoffRelocationIterator`, or is that ok?

Reply:
The `CoffRelocationIterator` change is good.

Reply:
Ok, removed.